libs/androidlib/src/mill/androidlib/AndroidR8AppModule.scala

```diff
@@ -60,19 +59,25 @@ trait AndroidR8AppModule extends AndroidAppModule {
       .flatMap(_.proguardRules)
   }
 
+  /**
+   * The SubPaths for the proguard files
+   * in this project's sources.
+   * @return
+   */
+  def androidProjectProguardFiles: Seq[os.SubPath] = Seq.empty[os.SubPath]
+
+  /**
+   * Defines the sources for the proguard files.
+   * By default, uses the SubPaths from [[androidProjectProguardFiles]].
+   * @return
+   */
+  def androidProjectProguardSources: T[Seq[PathRef]] = Task.Sources(
+    androidProjectProguardFiles*
+  )
```

Comment:
What is the motivation for the two-part API? Did you hit any issues with just overriding `androidProjectProguardSources` directly (which I would prefer), if you want to change the file set?

`Task.Sources` already accepts sub-paths (in addition to `Path`s), so it should be equally convenient to write.